src/lib/draw/image/ImageLayer.svelte

```diff
@@ -0,0 +1,96 @@
+<script lang="ts">
+  import type { ImageSource } from "maplibre-gl";
+  import { map, mode } from "stores";
+  import { onDestroy } from "svelte";
+  import { Marker } from "svelte-maplibre";
+  import { imgSrc, opacity } from "./stores";
+
+  // TODO Upstream an ImageLayer component
```

Comment:
Most of the code here can go away soon. I'll add something to svelte-maplibre to wrap up the reactivity properly